### PR TITLE
Task-43073: Implement a rest method to check if the current user is delegated administrator or not

### PR DESCRIPTION
**Pull request by AzmiTouil**

### CODE REVIEW

File: component/portal/src/main/java/org/exoplatform/portal/rest/UserRestResourcesV1.java [hunk 1]
```diff
@@ -510,6 +510,26 @@ public Response isSuperUser() {
...
+          response = Response.class
+  )
+  @ApiResponses(
+          value = {
+                  @ApiResponse(code = 200, message = "Request fulfilled"),
+                  @ApiResponse(code = 500, message = "Internal server error due to data encoding"),
+          }
+  )
+  public Response isDelegatedAdministrator() {
+    boolean isDelegatedAdministrator = userACL.isUserInGroup("/platform/delegated") && !userACL.isUserInGroup("/platform/administrators");
```
Comment (azayati) [inline component/portal/src/main/java/org/exoplatform/portal/rest/UserRestResourcesV1.java]: is it possible to put "/platform/delegated" and "/platform/administrators" as constants ?

**Review by azayati — CHANGES_REQUESTED** [review]
See review

File: component/portal/src/main/java/org/exoplatform/portal/rest/UserRestResourcesV1.java (hunk 1 above)
Comment (AzmiTouil) [reply component/portal/src/main/java/org/exoplatform/portal/rest/UserRestResourcesV1.java]: yes, done !

**Review by azayati — APPROVED** [review]
LGTM after QA validation

**Review by rdenarie — APPROVED** [review]

**Pull request merged**